### PR TITLE
3353 handle persisted notifications

### LINKED ISSUE #3353: Handle new notifications system
Integrate with backend notification api to handle notification events received while app is not live.

### PR DESCRIPTION
**Pull request by jryu01**
# Description

- Integrated with server notification endpoints to pull persisted and clear notifications
- Some style updates and refactoring of existing code

Resolves #3353

### CODE REVIEW

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java [hunk 1]
```diff
@@ -73,32 +73,40 @@ public class ExtractionService {
 	// time the progress takes to reach each subsequent half
 	final Double HALFTIME_SECONDS = 2.0;
 
+	// @Value("${terarium.extractionService.poolSize:10}")
```
Comment (dvince2) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java:76]: This property is still in application.properties - you should be able to use this and get rid of the hard coded pool size.

**Review by dvince2 — COMMENTED** [review]
back end looks good - just one comment on application.props

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java (hunk 1 above)
Comment (jryu01) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java]: I updated it. I think the changes you made in my previous PR are lost so I added it back.

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java [hunk 2]
```diff
@@ -73,32 +73,40 @@ public class ExtractionService {
...
+	private int POOL_SIZE = 10;
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java]: I think some changes were lost in main branch that are then merged into my PR. I updated and added back lost changes.

File: packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue [hunk 3]
```diff
@@ -107,6 +111,31 @@ const getElapsedTimeText = (item: NotificationItem) => {
 </script>
 
 <style>
+/*
+ * Reset the default overlay component container style.
+ * Note that this style block isn't scoped since the overlay component is appended to the body html dynamically when opened
+ * and is placed outside of this component's scope and the scoped styles aren't applied to it.
+ */
+.notification-panel-container.p-overlaypanel {
+	top: 51px !important;
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue]: Done

File: packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue [hunk 4]
```diff
@@ -107,6 +111,31 @@ const getElapsedTimeText = (item: NotificationItem) => {
...
+	width: 540px;
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue]: Done

File: packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue [hunk 5]
```diff
@@ -107,6 +111,31 @@ const getElapsedTimeText = (item: NotificationItem) => {
...
+	box-shadow: 0px 4px 4px 0px #00000040;
+	padding: var(--content-padding);
+	padding-bottom: 1.5rem;
+	gap: var(--gap);
+	border: 1px solid #c3ccd6;
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue]: Done

File: packages/client/hmi-client/src/services/notificationEventHandlers.ts
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/services/notificationEventHandlers.ts:1]: Prettier?

**Review by YohannParis — APPROVED** [review]

**Pull request merged**